modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,55 @@
+|Runs on any CPU not used by a low latency container, including reserved
+
+|Infrastructure pods
+|Runs on any CPU not used by a low latency container, including reserved
+
+|Interrupts
+|Redirected to reserved CPUs (optional in {product-version} and later)
+
+|Kernel processes
+|Pinned to reserved CPUs
+
+|Latency sensitive workload pods
+|Pinned to a specific set of exclusive CPUs from the isolated pool
+
+|OS processes/systemd services
+|Pinned to reserved CPUs
+|===
+
+The exact partitioning pattern to use depends on many factors like hardware, workload characteristics and the expected system load. Some sample use cases are as follows:
+
+* If the latency sensitive workload uses specific hardware (like network interface cards (NICs)), ensure that the CPUs in the isolated pool are as "close" to this hardware as possible. Place the workload in at least the same Non-Uniform Memory Access (NUMA) node.
+
+* The reserved pool is used for handling all interrupts. When depending on system networking, allocate a sufficiently sized reserve pool to handle all the incoming packet interrupts. In {product-version} and later, workloads can optionally be labeled as sensitive. For more information, see the Red Hat Knowledgebase solution xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus[Managing device interrupt processing for guaranteed pod isolated CPUs].
+
+The decision regarding which specific CPUs should be used for reserved and isolated partitions requires a detailed analysis and measurements. Factors like NUMA affinity of devices and memory play a role. The selection also depends on the workload architecture and the specific use case.
+
+[IMPORTANT]
+====
+The reserved and isolated CPU pools must not overlap and together must span all available cores in the worker node.
+====
+
+To ensure that housekeeping tasks and workloads do not interfere with each other, specify two groups of CPUs in the `spec` section of the performance profile. 
 
 * `isolated` - The CPUs for the application container workloads. These CPUs have the lowest latency. Processes in this group have no interruptions and can, for example, reach much higher DPDK zero packet loss bandwidth.
 
-* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group tend to be very busy, so latency-sensitive applications should be run in the `isolated` group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
+* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group are often busy, so latency-sensitive applications should be run in the `isolated` group.
```

Comment:
```suggestion
* `reserved` - The CPUs for the cluster and operating system housekeeping duties. Threads in the `reserved` group are often busy, so latency-sensitive applications should be run in the `isolated` group.
```

The infra containers are actually running and accounting towards the isolated pool. That they can physically execute on the reserved cores is a bit confusing and just an internal mechanics that does not change much.

Reply:
Updated as suggested.